tests/cases/fourslash/codeFixUnusedIdentifier_destructure_allUnused.ts

```diff
@@ -4,7 +4,7 @@
 // @noUnusedParameters: true
 
 ////export {};
-////const { x, y } = o;
+////const { x, y } = [{}];

```

Comment:
Why this change? This makes the code wrong since it destructures an array to an object..

Reply:
Fixed, accidentally provided an array rather than an object.